### ISSUE TITLE
Unittests for widget organization_list.dart

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#1294]
We will need unittests done for all methods, classes and functions found in this file.

Any widgets referenced in this file must also have unittests done

```
organization_list.dart 
```

This file can be found in the `lib/widgets` directory

Parent Issue: 
- https://github.com/PalisadoesFoundation/talawa/issues/1125

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa/tree/develop/lib/widgets)
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide

**Comment by vedantkulkarni** [comment]
Hi can I work on this one?

**Comment by Ayush0Chaudhary** [comment]
Can you assign this issue to me @palisadoes ??
As no activity from vedant

**Comment by peanutbutter003** [comment]
@palisadoes can you assign this issue to me??

**Comment by noman2002** [comment]
Unassigned due to inactivity.

**Comment by nidhin29** [comment]
Can you assign this to me

**Comment by palisadoes** [comment]
As mentioned in Slack, this week contributor @aashimawadhwa is handling all new issue assignments and seeking reviewers for PRs. Please work with them to coordinate these activities.

**Comment by nidhin29** [comment]
@aashimawadhna  ma'am can you assign this to me

**Comment by nidhin29** [comment]
@aashimawadhwa ma'am can you assign this to me

**Comment by nidhin29** [comment]
No response

**Comment by aashimawadhwa** [comment]
assigned to you @nidhin29

**Comment by nidhin29** [comment]
Sorry I am not able to do it

**Comment by gpkonarde** [comment]
@aashimawadhwa Kindly assign this issue to me

**Comment by Manik2708** [comment]
Can I work on this issue @Cioppolo14?

**Comment by Azad99-9** [comment]
@Manik2708 for testing purpose we use hard-coded mock data. Nothing comes from API.

**Comment by Azad99-9** [comment]
Will it come from API side?

**Comment by Manik2708** [comment]
That's what I am asking!

**Comment by Azad99-9** [comment]
I suggest you to ask mentors.

**Comment by Manik2708** [comment]
@noman2002 @Ayush0Chaudhary @literalEval I am suffering from a problem when testing the `Query Hook Widget`, the mock query is not getting called inside this widget when following the same approach as in `fetch_plugin_list_test.dart`, I am getting nothing on searching for writing tests including this widget. Can I get reason for using this widget in this file and not invoking the query directly? As this `Query` widget is used only in this file!

**Comment by Manik2708** [comment]
I was trying for this from last two weeks, but was not able to mock the behaviour of `Query` widget.There is a reason for this and so, I have raised an issue regarding this in GraphQL, @literalEval @noman2002 please see this issue and see if it can be fixed, as I am not able to get it's solution.

**Comment by literalEval** [comment]
Hey @Manik2708 is the mock for Query widget generated ?

**Comment by Manik2708** [comment]
No, please refer to the issue https://github.com/zino-hofmann/graphql-flutter/issues/1408 There is a problem in sending mock query. Let me explain my approach, please see if something wrong.
1) There is a test written related to GraphQL query, `plugins/fetch_,plugins_list.dart`. I am following the same approach.
2) Mocking of client is successful but sending a query is failing. `result.data` is always coming null.
3) I went to the API reference of this `Query` widget, it's a widget which is extending `HookWidget`
4) The method which is being used for fetching query in this widget is not wrapped in any class, hence I am not able to mock it. This method is directly declared and implemented in the file. I searched a lot on Google and Stack Overflow but found nothing. Hence I raised an issue in the package, so that I could find a way of mocking it or they could fix it if it's a big!

**Comment by Manik2708** [comment]
I think this is why most of assignees were not able to complete this issue. I maybe wrong. Please verify my approach! Thanks.

**Comment by noman2002** [comment]
@Manik2708 Thank you for letting us know the exact problem, your approach is fine. Let's figure out some alternates for this.

**Comment by Manik2708** [comment]
Till then unassigning myself.

**Comment by Cioppolo14** [comment]
@Cyclotron17 Are you working on this?

**Comment by palisadoes** [comment]
unassigning, inactivity

**Comment by syedali237** [comment]
Can I be assigned this issue?

**Comment by palisadoes** [comment]
unassigning. Inactivity

**Comment by Dante291** [comment]
@ARYPROGRAMMER is this issue resolved?

**Comment by ARYPROGRAMMER** [comment]
> @ARYPROGRAMMER is this issue resolved?

This is currently untouched and the coverage remains at 60.34%. Would raise a PR soon to fix this as well.

**Comment by ARYPROGRAMMER** [comment]
@palisadoes can you assign this to me

**Comment by ARYPROGRAMMER** [comment]
@palisadoes While i was working on this, there are multiple issues that I found out. The GraphQL mock generator seems to break for the new tests I was writing. Will update when done with that

**Comment by palisadoes** [comment]
Thanks. We need to get this test completed for the stability of the app.

**Comment by palisadoes** [comment]
@ARYPROGRAMMER 

Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

**Comment by ARYPROGRAMMER** [comment]
> @ARYPROGRAMMER
> 
> Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

Surely, would need time. Thanks

**Comment by may-tas** [comment]
Please assign me this issue.

**Comment by may-tas** [comment]
Working on it.

**Comment by palisadoes** [comment]
Closing, inactivity

**Comment by may-tas** [comment]
@palisadoes Sorry for the delay, I was busy due to my university exams.
I am still working on this.

**Comment by Cioppolo14** [comment]
Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue

**Comment by MohitMaulekhi** [comment]
Pls assign

**Comment by palisadoes** [comment]
Unassigned. Inactivity

**Comment by MohitMaulekhi** [comment]
I was busy with something, working on it pls reassign

**Comment by aikansh008** [comment]
Hello sir can you please assign these issue to me.

**Comment by mpdesai90** [comment]
Please don't ask to assign the issue if it has been already assigned.

**Comment by Cioppolo14** [comment]
Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

**Comment by Naman2004singh** [comment]
Hey, Could you kindly assign me to this issue?

**Comment by palisadoes** [comment]
unassigning inactivity

**Comment by Mattboss10** [comment]
Can I work on this issue?

**Comment by Cioppolo14** [comment]
Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue

**Comment by vamshi567bh** [comment]
@palisadoes please assign me

**Comment by palisadoes** [comment]
@BharathC0

You already have >=2 issues assigned to you. Our policy is 2 issues across all repositories. When you have merged the PRs related to your issues, then we can consider assignments. 

Please refrain from asking to be assigned. You have requested to be assigned to dozens of issues that requires us to look at the email, validate the number of issues assigned and then leave this message. Please be considerate of our time.
